### ISSUE TITLE
Clarification on pretrained checkpoints

### ISSUE THREAD

**Issue by yeshwanth95** [Daisy-1227/FRI-Net#3]
Hello! Thank you for open-sourcing your amazing work. I've been finding your paper and code extremely useful in my ongoing projects.

I have a quick question regarding the pretrained checkpoints used in the resnet50 backbone in your paper. You mention that you use the checkpoints of the RoomFormer method to initialize the backbone during `phase0` of training here:
https://github.com/Daisy-1227/FRI-Net/blob/560d1c5e1dae114da061ccb36699e2bcb2d8266e/train_stru3d.py#L122C1-L125C12

1. Is there a reason for this besides the better performance? Did you also try training from scratch and what was the performance differential between the two cases?
2. Are the numbers reported in the paper from experiments using the Roomformer checkpoints?

I would like to know if this could potentially have implications in generalization when training/testing on datasets other than Structured3D and SceneCAD. I'm also working on running the experiments myself and figuring out the difference in performance between the two approaches. However, I would like to get your insights before doing so. Looking forward to your reply. Thank you.

**Comment by Daisy-1227** [comment]
Hi! Thanks for being interested in our work!

1. The use of a pretrained checkpoint is primarily aimed at improving convergence during the training process. By leveraging a pretrained backbone, the model gains prior knowledge of room-wise features, which stabilizes the matching between predicted rooms and ground truth (GT) rooms. This is particularly critical as the loss function relies on bipartite matching to align predicted rooms with GT rooms. Consequently, this trick enhances the overall performance of the model.

2. The model metrics reported in our paper are based on the Roomformer checkpoint pretraining.
